apps/site/assets/ts/shuttles/components/ShuttlesOverview.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { ReactElement, useState, useEffect } from "react";
+
+interface Props {
+  places: {
+    0: string;
+    1: string;
+  };
+}
+
+const ShuttlesOverview = ({ places }: Props): ReactElement<HTMLElement> => {
+  const [selectedDirectionId, setSelectedDirectionId] = useState("");
+  useEffect(() => {}, [selectedDirectionId]);
```

Comment:
@thecristen this `useEffect` doesn't seem to do anything

Reply:
true - it's intended as a placeholder that will handle whatever is needed to update the future overview map.

Reply:
OK, in that case let's add a comment to that effect.